react-hook-form-mui/src/forms/Select.js

```diff
@@ -26,21 +24,23 @@ const Select = ({
 				multiple,
 				native: isNative,
 			}}
+			multiple={multiple}
+			select
 		>
 			{isNative
 				? options.map((option, i) => (
-					<option key={i} value={option.value}>
-						{option.label}
+					<option key={i} value={option.code}>
+						{option.name}
 					</option>
 				))
 				: options.map((option, i) => (
-					<MenuItem key={i} value={option.value}>
-						{option.label}
+					<MenuItem key={i} value={option.code}>
+						{option.name}
```

Comment:
Why is it option.name and not label? we would be sending options as a pair of key value pairs of the form of [{value:" " ,label: " "},{value:" ",label:" "}]